### DISCUSSION TITLE
Where does static page data come from?

### DISCUSSION THREAD

**Discussion by aarslantas** [vercel/next.js#71532] [Help] [answered]
### Summary

I have a project that I created with Nextjs 14 and I have an about page in this project. There is a component called infoComponent in the about page. I have 2 questions about this.

1 - When I go to the about page from the home page at the beginning, there are two requests in the Network tab. The text("hey info container") in the info component is not in their response. Where exactly does this text come from? Even if it was already created during the build, shouldn't I see it in the network when I go to /about? I also looked in all the js files that came when the application was first loaded. I couldn't see this text("hey info container") there either.

2 - Again, the second of these two requests sometimes comes empty. Why?

![github1](https://github.com/user-attachments/assets/f661c667-a424-41b5-8d3c-37ee1dac0eea)

![github2](https://github.com/user-attachments/assets/5ec868f6-e538-4c2b-94e1-9838b13d1217)

![github3](https://github.com/user-attachments/assets/34d94a23-bfaa-49a8-a9be-8bb89fbe7996)

![github5](https://github.com/user-attachments/assets/d9005b10-2cc8-4bc0-80c2-47b5a6991afd)

**Comment by icyJoseph** [comment, marked as answer]
Are you sure about this? Try it with a production build. Upon loading `/`, IIRC, the `/about`, route is prefetched, assuming you have `next/link`, and then you should see one of those `rsc` requests, and in their response there should be something like (example):

```text
0:["development",[["children","(main)","children","__PAGE__",["__PAGE__",{}],"$L1",[null,"$L2"]]]]
4:I["(app-pages-browser)/./app/(main)/components/Pronunciation.tsx",["app/(main)/page","static/chunks/app/(main)/page.js"],"Pronunciation"]
5:I["(app-pages-browser)/./node_modules/next/dist/client/link.js",["app/(main)/not-found","static/chunks/app/(main)/not-found.js"],""]
6:"$Sreact.suspense"
```

Give it a go. Then you can use https://rsc-parser.vercel.app/ to interpret those results.

**Reply by icyJoseph** [reply]
And as for, 

> Again, the second of these two requests sometimes comes empty. Why?

That's not true, necessarily, it's just not been consumed. Try it yourself. Make a request using fetch, but don't call .JSON (or .text) and then look for it on the network tab.
